pipeline-scripts/release.groovy

```diff
@@ -10,7 +10,9 @@ buildlib = load("pipeline-scripts/buildlib.groovy")
 commonlib = buildlib.commonlib
 slacklib = commonlib.slacklib
 
-oc_cmd = "oc --kubeconfig=${buildlib.ciKubeconfig}"
+buildlib.withAppCiAsArtPublish() {
+    oc_cmd = "oc --kubeconfig=${KUBECONFIG}"
```

Comment:
After the closure returns, `KUBECONFIG` will no longer exist. Needs to have the actual command execution enclosed.

Reply:
Updated